.github/workflows/build-ci.yml

```diff
@@ -31,6 +31,12 @@ jobs:
                       laravel: "10.*"
                       mongodb: "5.0"
                       mode: "low-deps"
+                      os: "ubuntu-latest"
+                    - php: "8.4"
```

Comment:
Do you want to allow failures on 8.4, or wait until https://github.com/spatie/laravel-ray/pull/340 is merged?

Reply:
I'll see tomorrow, depending on how responsive the maintainers are.

Reply:
Waiting for the branch alias to be fixed https://github.com/spatie/laravel-ray/pull/341, or the fix to be released.

Reply:
New package update: https://github.com/spatie/ray/pull/903